text/000-rustdoc-jump-to-definition.md

```diff
@@ -0,0 +1,178 @@
+Rustdoc: jump to definition
+
+- Feature Name: `jump_to_definition`
+- Start Date: 2021-12-09
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#89095](https://github.com/rust-lang/rust/issues/89095)
+
+# Summary
+[summary]: #summary
+
+Generate links on idents in the rustdoc source code pages which links to the item's definition and documentation (if available).
+
+# Motivation
```

Comment:
issue: the contents of this section are much much more than just "motivation", i think this rfc needs to be organized a bit more